src/source/amplify.rs

```diff
@@ -14,7 +14,15 @@ where
 
 /// Internal function that converts decibels to linear
 pub(super) fn to_linear(decibels: f32) -> f32 {
```

Comment:
Should we move these methods to `mod.rs` or some utility module?

Reply:
I moved and renamed them to `src/math.rs` in b1497f8. It doesn't feel right to me that we were adding a utility function `to_db` to a module that didn't use it. I also made them public, because they are useful as general-purpose audio functions.